marketplace-core/pom.xml

```diff
@@ -97,6 +97,10 @@
       <groupId>org.webjars.bower</groupId>
       <artifactId>angular-ui-bootstrap-bower</artifactId>
     </dependency>
+    <dependency>
```

Comment:
@pamval, in order to do not duplicate the code. I move all logic to kettle-core: https://github.com/pentaho/pentaho-kettle/blob/09bb45c0cfefebfdd214acd94f1d8651f697fec1/core/src/main/java/org/pentaho/di/core/util/HttpClientManager.java and https://github.com/pentaho/pentaho-kettle/blob/09bb45c0cfefebfdd214acd94f1d8651f697fec1/core/src/main/java/org/pentaho/di/core/util/HttpClientUtil.java